index.js

```diff
@@ -282,6 +286,10 @@ const registerEvents = () => {
 };
 
 export const config = plugin.config = {
+	validateOnlyWhenConfigured: {
```

Comment:
linter-jscs has the same config
https://github.com/AtomLinter/linter-jscs/blob/master/index.js#L143
they call it `onlyConfig`